src/js/puzzles.js

```diff
@@ -54,6 +54,34 @@ var puzzlesArray = [
     word: 'space',
     hint: 'competition',
     answer: 'race'
+  },
+  {
+    id: 9,
+    sentence: 'I wrote on the paper with my men',
```

Comment:
This sentence makes enough sense that it may be difficult to interpret where the illogical point is until the hint is displayed. Can you switch out the key word with another that makes it a little more ridiculous? Ten? Fin?